src/Extension.php

```diff
@@ -52,8 +53,26 @@ public function getConfigKey()
 
     public function initialize(ExtensionManager $extensionManager)
     {
-        // PHPUnit
-        require_once BASE_PATH . '/vendor/phpunit/phpunit/src/Framework/Assert/Functions.php';
+        // Find PHPUnit assertion implementations
+
+        $found = false;
+
+        $options = [
+            BASE_PATH . '/vendor/phpunit/phpunit/src/Framework/Assert/Functions.php',
+            BASE_PATH . '/vendor/sminnee/phpunit/src/Framework/Assert/Functions.php',
+        ];
```

Comment:
@dnsl48 Won't `sminnee/phpunit` always be installed now that it's an explicit requirement of this module?

Reply:
Fair enough. Although, if we update composer.json in the future versions of behat-extension and would use e.g. phpunit 9 or something like that, this code will keep working transparently.

Reply:
Okay, I'll settle for shifting `sminnee` up so it saves a loop in the meantime 😛

Reply:
```suggestion
        $options = [
            BASE_PATH . '/vendor/sminnee/phpunit/src/Framework/Assert/Functions.php',
            BASE_PATH . '/vendor/phpunit/phpunit/src/Framework/Assert/Functions.php',
        ];
```